core/iwasm/interpreter/wasm_opcode.h

```diff
@@ -1021,7 +1038,7 @@ typedef enum WASMAtomicEXTOpcode {
         SET_GOTO_TABLE_ELEM(WASM_OP_MISC_PREFIX),    /* 0xfc */ \
         SET_GOTO_TABLE_SIMD_PREFIX_ELEM()            /* 0xfd */ \
         SET_GOTO_TABLE_ELEM(WASM_OP_ATOMIC_PREFIX),  /* 0xfe */ \
-        DEF_DEBUG_BREAK_HANDLE()                                \
+        DEF_DEBUG_BREAK_HANDLE() DEF_EXT_V128_HANDLE()          \
```

Comment:
Had better put `DEF_EXT_V128_HANDLE()` in a single line

Reply:
yeah I agree but this one is forcefully corrected by clang-format so I guess I'll have to leave it like that

Reply:
OK, got it, thanks.